code-challenges/ArrayReverse/ArrayReverse.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Arrays;
+
+public class ArrayReverse {
+  public static void main (String[] args) {
+    // create the array to test your code on later
+    int[] startArr = new int[] {1,2,3,4,5,10};
+    // print it out, nicely
+    System.out.println(Arrays.toString(startArr));
+    // call your reverseArray method and save the result in a variable
+    int[] endArr = reverseArray(startArr);
+    // print out the result, nicely
+    System.out.println(Arrays.toString(endArr));
+  }
+
+  // the method you should write, to reverse an array
+  public static int[] reverseArray(int[] inputArray) {
+    int [] shrimp = new int [inputArray.length];
+    int tigerPrawn = inputArray.length;
+    for (int i = 0; i < inputArray.length; i++) {
+      shrimp[tigerPrawn - 1] = inputArray[i];
```

Comment:
This works perfectly! Just some seafood for thought: instead of creating `tigerPrawn` and then decrementing it, could you have just subtracted `i`?